packages/cli/src/rpc.ts

```diff
@@ -121,6 +105,32 @@ export class RPCManager {
       this.log(`TEST PONG received from ${encodedENR.nodeId}`)
       return `PING/PONG successful with ${encodedENR.nodeId}`
     },
+    portal_history_findContent: async (params: [string, Uint8Array]) => {
+      const [enr, contentKey] = params
+      await this._client.sendFindContent(enr, contentKey, SubNetworkIds.HistoryNetwork)
```

Comment:
```suggestion
     const res =  await this._client.sendFindContent(enr, contentKey, SubNetworkIds.HistoryNetwork)
     return res
```
Can we do something like this to return the response from this message?  I don't remember exactly what it looks like so we may need to decode it or something but we should always return some sort of response from RPC methods (even if it's only `request succeeded` or something like that)